tests/data/placebo/test_datapipeline_reporting/datapipeline.DescribePipelines_1.json

```diff
@@ -0,0 +1,82 @@
+{
+    "status_code": 200, 
+    "data": {
+        "ResponseMetadata": {
+            "RetryAttempts": 0, 
+            "HTTPStatusCode": 200, 
+            "RequestId": "75644193-07e1-11e7-91e2-bfd5af170b83", 
+            "HTTPHeaders": {
+                "x-amzn-requestid": "75644193-07e1-11e7-91e2-bfd5af170b83", 
+                "date": "Mon, 13 Mar 2017 11:37:37 GMT", 
+                "content-length": "849", 
+                "content-type": "application/x-amz-json-1.1"
+            }
+        }, 
+        "pipelineDescriptionList": [
+            {
+                "fields": [
```

Comment:
i think we should grab all the fields for augmenting the resource record at least those with an '@' prefix, strip the prefix and use for the augmentation, and then we can fill date model attribute in as well (creationDate over lastActivationDate).